woodwork/logical_types.py

```
@@ -64,6 +64,10 @@ def transform(self, series):
                 raise TypeConversionError(series, new_dtype, type(self))
         return series
 
+    def validate(self, series, return_indices=False):
+        if return_indices:
+            return pd.Series(name=series.name, dtype="boolean")
```

Alternatively, the default mask could be non-null values

---

Maybe the default behavior should return None